src/backend/distributed/sql/udfs/worker_split_shard_replication_setup/latest.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TYPE citus.split_shard_info AS (
```

Comment:
nitpick : Add comments for new Types and Functions.

Reply:
Done